providers.tf

```diff
@@ -1,18 +1,6 @@
 terraform {
   required_version = "0.13.7"
   required_providers {
-    aws = {
-      source  = "hashicorp/aws"
-      version = "~> 3"
-    }
-  }
-}
-
-provider "aws" {
-  region = var.region
-  default_tags {
-    tags = {
-      "deployment" = "sourcegraph-executors"
-    }
+    aws = "~> 3.0"
   }
```

Comment:
can we move this default_tags part into our config in infra, if not yet happened? It'll help us keep track of the cost better.